_emails/eureka.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Eureka
+permalink: "/eureka"
+name: Letter to City Council
+state: CA
+city: Eureka
+layout: email
+recipients:
+- sseaman@ci.eureka.ca.gov
+- lcastellano@ci.eureka.ca.gov
+- hmessner@ci.eureka.ca.gov
+- kbergel@ci.eureka.ca.gov
+- aallison@ci.eureka.ca.gov
+- narroyo@ci.eureka.ca.gov
+- cityclerk@ci.eureka.ca.gov
+subject: Commit to reallocate for social equity
+body: |-
+To whom it may concern,
+
+I am a resident of Eureka's [YOUR WARD] Ward. I am writing to demand that the City Council adopt a budget strategy that prioritizes community well-being and redirects funding away from the police in the next budget evaluation period.
+
+We have seen mounting evidence that police departments are ineffective institutions that marginalize minority communities and put citizens at risk of injury and death, yet the police budget accounts for 46% of our general fund (see this article). I ask that you redirect the majority of the $14.2M allotted for crime prevention toward community programs that provide citizens with basic human needs, like affordable healthcare and housing. We don’t need a militarized police force. We need to create a space in which more mental health service providers, social workers, victim/survivor advocates, religious leaders, neighbors, and friends - all of the people who really make up our community - can look out for one another. This is of course a long transition process, but real, actionable change starts with reallocating funding and investing in inclusive and diverse support strategies for our community.
+
+As the City Council, the budget proposal is in your hands. It is your duty to represent your constituents. I am urging you to completely revise the budget for the 2020-2021 fiscal year. We can be a beacon for other cities to follow if only we have the courage to change.
+
+Thank you for your time,
+
+[YOUR NAME]
+[YOUR ADDRESS]
+[YOUR PHONE NUMBER]
+[YOUR EMAIL ADDRESS]
```

Comment:
This body text (under the pipe, which should be |-) should be indented.

Reply:
Ok. will do now and for my other PRs